templates/Element/FilterBox/filter_box_common.twig

```diff
@@ -101,20 +101,22 @@
 
             <span v-if="filter.name === 'categories'">
                 <label>{{ __('Categories') }}</label>
-
                 {% if schemasByType %}
                     {% for type, typeSchema in schemasByType %}
-                        <category-picker
-                            v-if="queryFilter.filter.type == '{{ type }}'"
-                            :categories="{{ typeSchema.categories|json_encode }}"
-                            :initial-categories="initCategories"
-                            id="category-filter"
-                            form="_filters"
-                            label=""
-                            @change="onCategoryChange"
-                        ></category-picker>
+                        {% if typeSchema.categories %}
```

Comment:
qui e in altri punti più in basso (anche per initCategories) si controlla solo se ci siano, ha senso anche controllare che non siano vuote le categorie? o lasciamo così? valuta pure tu

Reply:
Lo schema ha oppure no le categorie, non dovrebbe mai avere categorie vuote (se un modello non avrà mai categorie, togliamo l'association categories dal model, in genere). Lascerei così